viz.yaml

```diff
@@ -360,5 +359,32 @@ publish:
     template: vertical-social-media-template
     publisher: social
     depends: social-links
-
+    context:
+      embedLink: national-water-use.html
```

Comment:
do we want that to be "national-water-use" or "2015-water-use"?

Reply:
Let's see...the base url is going to be:
https://owi.usgs.gov/vizlab/water-use-15
maybe we want something more simple like:
https://owi.usgs.gov/vizlab/water-use-15/embed.html
?
I guess I don't have a great opinion one way or another...for reference we did it in Irma like this:
https://owi.usgs.gov/vizlab/hurricane-irma/embed-hurricane-irma.html

Reply:
hmmm OK, let's use just `embed` then? Would the hashes end up before or after the word?

Reply:
or embed-water-use-15?

Reply:
https://owi.usgs.gov/vizlab/water-use-15/embed-water-use-15.html
?
I'll push that up in a sec......unless I hear otherwise in the meantime....

Reply:
the hashes _should_ end up after the word. say something if they don't.

either embed-water-use-15.html or embed.html works for me.

Reply:
Yeah, they do. So, if you type into your browser:
https://owi.usgs.gov/vizlab/water-use-15/embed-water-use-15.html
It automatically adds the hash stuff.
We can also directly add the hash stuff to a particular state/category for a particular news outlet. So....I'm pretty sure we're all on the same page...I just *wasn't* sure we would be before I opened this PR.